src/Tester.js

```diff
@@ -98,14 +107,20 @@ Tester.propTypes = {
   store: PropTypes.instanceOf(TestHookStore),
   specs: PropTypes.arrayOf(PropTypes.func),
   waitTime: PropTypes.number,
-  clearAsyncStorage: PropTypes.bool
+  clearAsyncStorage: PropTypes.bool,
+  testStartDelay: PropTypes.number,
+  consoleLog: PropTypes.oneOfType([PropTypes.bool, PropTypes.string]),
+  reporter: PropTypes.string,
```

Comment:
Will address in the reporting PR to follow https://github.com/pixielabs/cavy/pull/21